Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,564 @@
+# Frequenty asked questions
+
+## General information and availability
+
+### Do I need VTK to build ITK ?
+
+With the default configuration, ITK does not require VTK. However, some
+optional modules (e.g. [ITKVTKGlue](https://github.com/InsightSoftwareConsortium/ITKVtkGlue)) do require VTK.
+
+### Data formats
+
+#### What 3D data file formats can ITK import and export?
+
+The following table identifies the file formats that ITK can read/write
+via a factory mechanism based on the file extension
+([`itk::ImageIOFactory`](https://itk.org/Doxygen/html/classitk_1_1ImageIOFactory.html)). Some proprietary files format are only imported.
+
+More information on supported file formats is available on the
+[wiki ITK File Formats](https://itk.org/Wiki/ITK_File_Formats) and [ITK-wasm Image File Formats](https://wasm.itk.org/en/latest/introduction/file_formats/images.html)
```

Comment:
I think the wiki link is outdated and should be removed.

Reply:
Removed in fe3d292.